internal/db/schema/migrations/oss/postgres/80/01_storage_bucket.up.sql

```diff
@@ -0,0 +1,11 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+
+drop trigger if exists immutable_columns ON storage_plugin_storage_bucket;
```

Comment:
```suggestion
drop trigger immutable_columns on storage_plugin_storage_bucket;
```
The trigger should exist and ff it doesn't we want to error out.